auslib/blobs/apprelease.py

```diff
@@ -908,6 +916,61 @@ def getReferencedReleases(self):
         return referencedReleases
 
 
+class AdditionalPatchAttributesXMLMixin(object):
+
+    def _getAdditionalPatchAttributes(self, patch):
+        additionalPatchAttributes = {'proof': ''}
+
+        if 'proof' in patch:
+            additionalPatchAttributes['proof'] = ' proof="%s"' % patch["proof"]
+
+        return additionalPatchAttributes
+
+
+class ReleaseBlobV8(AdditionalPatchAttributesXMLMixin, ReleaseBlobBase, NewStyleVersionsMixin, MultipleUpdatesXMLMixin, UnifiedFileUrlsMixin):
+    """
+
+    Changes from ReleaseBlobV7:
+        * Adds support for ProofXMLMixin (ProofXMLMixin is placed as first parameter for inheritance preference)
+
+    For further information:
+        * https://bugzilla.mozilla.org/show_bug.cgi?id=1384296
+
+    """
+    jsonschema = "apprelease-v8.yml"
+
+    # for the benefit of get*XML
+    optional_ = ('showPrompt', 'showNeverForVersion',
+                 'actions', 'openURL', 'notificationURL', 'alertURL',
+                 'promptWaitTime', 'backgroundInterval')
```

Comment:
I seem to recall something about needing to adding something here as well. If so, feel free to that in this PR or in a follow-up - your choice.